features/google/src/views/ministories/story.jsx

```diff
@@ -4,7 +4,7 @@ class Story extends Component {
     constructor({ account, mode }) {
         super();
         this.analyses = account.analyses;
-        this.neededAnalyses = [];
+        this._neededAnalyses = [];
```

Comment:
This is (mostly) unrelated, right?

Reply:
No i mentioned it in the comment "`Making it so when analyses have no data they are actually not shown (through missing _ active was always true).`"

Reply:
I hope it's okay as it's a really really small change